spec/components/schema/errors.yaml

```diff
@@ -1,10 +1,16 @@
 openapi: 3.0.2
-paths: {}
+paths: { }
 info:
   title: Errors Responses
   version: 1.0.0
 components:
   responses:
+    ErrorResponse:
+      description: HTTP 400, 401, 404 have the same response format.
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Error"
```

Comment:
I think this was introduced because the 4XX was not considered a valid name in openapi validation